test/components/PaymentForm_spec.js

```diff
@@ -0,0 +1,95 @@
+import { expect } from 'chai';
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import $ from '../../src/money';
+import PaymentFormComponent from '../../src/components/PaymentForm';
+
+const {
+  renderIntoDocument,
+  findRenderedDOMComponentWithClass,
+  Simulate
+} = TestUtils;
+const PaymentForm = React.createFactory(PaymentFormComponent);
+
+function setup({ cash = 0, credit = 0, tip = 0 } = {}) {
+  const startingBalance = 2000;
+  const setClosed = () => {};
+  const component = renderIntoDocument(PaymentForm({
+    startingBalance,
+    cash,
+    credit,
+    tip,
+    setClosed
+  }));
+
+  return {
+    component,
+    startingBalance
+  };
+}
+
+describe('PaymentForm', () => {
+  const setClosed = () => {};
+
+  it('renders balance remaining after entered amounts', () => {
+    const { component, startingBalance } = setup();
+    const balance = findRenderedDOMComponentWithClass(component, 'payment-balance-amount');
```

Comment:
Since this is being used in all the tests below, it can also be in setup. same for input.